resolwe_bio/docker_images/chipseq/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+LABEL MAINTAINER Resolwe-bio authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common python python-dev python-pip && \
+
+    add-apt-repository -y ppa:fkrull/deadsnakes && \
+    apt-get update && \
+
+    apt-get -y install --no-install-recommends \
+      wget \
+      # required for bamliquidator
+      samtools \
+      libbam-dev \
+      libhdf5-serial-dev \
+      libboost-dev \
+      libboost-timer-dev \
+      libgoogle-perftools-dev \
+      libtbb-dev \
+      python-numpy \
```

Comment:
👍 on using distribution-provided versions of NumPy and SciPy. It also reduces image build time significantly.